tests/test_python/test_python.py

```diff
@@ -0,0 +1,42 @@
+from textwrap import dedent
+
+from dxr.testing import SingleFileTestCase
+
+
+class PythonSingleFileTestCase(SingleFileTestCase):
+    """Test case suited to testing the Python plugin."""
+    enabled_plugins = ('pygmentize', 'python')
```

Comment:
I usually go with mutability to determine what to use, rather than the contents; since I don't ever expect to change this value I went with tuple. I'm not necessarily _trying_ to stop people from changing it, but it feels like something that shouldn't change.

The pattern you mention is new to me, and interesting, but I feel like mutability is an easier rule. Not sure. (I mean it's moot for this PR anyway so yay).